appveyor.yml

```diff
@@ -3,9 +3,15 @@
 # license that can be found in the LICENSE file.
 
 install:
-  - ps: .\tool\appveyor.ps1 install
+  - cinst -y dart-sdk
+  - ps: $env:PATH = "$env:PATH;c:\tools\dart-sdk\bin"
+  - ps: $env:PATH = "$env:PATH;c:\Users\appveyor\AppData\Roaming\Pub\Cache\bin"
```

Comment:
(BTW, I'm not sure why Dart puts scripts under the roaming profile, as dart may not be available on all computers that a given user uses.)